backend/src/main/scala/cromwell/backend/validation/ContinueOnReturnCodeValidation.scala

```diff
@@ -0,0 +1,53 @@
+package cromwell.backend.validation
+
+import cromwell.backend.validation.RuntimeAttributesValidation._
+import cromwell.core._
+import wdl4s.types.{WdlArrayType, WdlStringType}
+import wdl4s.values.{WdlArray, WdlBoolean, WdlInteger, WdlString}
+
+import scalaz.Scalaz._
+
+object ContinueOnReturnCodeValidation {
+  val key = RuntimeAttributesKeys.ContinueOnReturnCodeKey
+
+  lazy val instance = new ContinueOnReturnCodeValidation {}
+
+  lazy val default = instance.withDefault(WdlInteger(0))
+
+  lazy val optional = default.optional
+
+  val missingMessage =
+    s"Expecting $key runtime attribute to be either a Boolean, a String 'true' or 'false', or an Array[Int]"
+}
+
+trait ContinueOnReturnCodeValidation extends RuntimeAttributesValidation[ContinueOnReturnCode] {
```

Comment:
super duper TOL, even for me, but the current trend is towards preferring abstract class unless you need a trait - e.g. http://stackoverflow.com/a/35251513

I'm trying to go this way myself, although I've said I'd never pick on someone in a code review. The only reason I'm making this comment was due to the way you phrased your response (which seemed to imply that abstract class was higher, instead of lower - or whichever direction one wants to go). I don't care if you change it or not.